src/models/db.js

```diff
@@ -0,0 +1,43 @@
+import pg from 'pg';
+
+const connection = {
```

Comment:
How do you intend to connect it with your production database?
Simply create a .env file, add your db connection strings there and use `dotenv` to access them

Reply:
I actually did, but forgot to remove the direct values. If you noticed, I commented out the environment connection.

Reply:
I will do that in the next commit. Can the pull request before merged now?

Reply:
Make changes and push it. Then, I'll merge it